include/swift/AST/Decl.h

```diff
@@ -5008,6 +5008,9 @@ class FuncDecl final : public AbstractFunctionDecl,
   }
 
   bool isGetter() const { return getAccessorKind() == AccessorKind::IsGetter; }
+  bool isComputedGetter() const {
```

Comment:
Hm, maybe just `hasExplicitAccessorKeyword()`, and not combine `isGetter` into it? Although I still think we should consider just going with the original version of your patch and not adding the fix-it even when there is an explicit `get`. @slavapestov?